docs/IWantTo.md

```diff
@@ -22,62 +22,27 @@ amounts of this token can have. E.g.
 You can also add a `toString` override, if you like.
 
 ```kotlin
-    class MyTokenType(override val tokenIdentifier: String, override val fractionDigits: Int = 0) : TokenType
+    val myTokenType: TokenType = TokenType(tokenIdentifier = "TEST", fractionDigits = 2)
 ```
 
-
 The `tokenIdentifier` is used along with the `tokenClass` property (defined
 in `TokenType`) when serializing token types. Two properties are required,
 as with `FiatCurrency` and `DigitalCurrency`, there can be many different
 instances of one `tokenClass`, each with their own `tokenIdentifier`.
 
-The above defined token type, allows CorDapp developers to create multiple
```

Comment:
we should keep this under another section which explains how to create your own token type sub-classes.